services/tasks-svc/internal/task-template/task_template.go

```diff
@@ -86,13 +102,29 @@ func (ServiceServer) CreateTaskTemplate(
 		Str("taskTemplateID", templateID.String()).
 		Msg("taskTemplate created")
 
+	// store event
+	if err := eventstoredb.SaveEntityEventForAggregate(ctx, s.es, NewTaskTemplateAggregate(templateID),
```

Comment:
A cleaner way is to 

- create a new EntityAggregateBase type, which embeds AggregateBase
- EntityAggregateBase then implements a custom Save() method, which does the same as SaveEntityEventForAggregate
- TaskTemplateAggregate, BedAggregate etc embed EntityAggregateBase instead of AggregateBase
- handler call `NewTaskTemplateAggregate(templateID).Save(...)`

Reply:
Yes. Sounds good. I've created a new issue for this.